src/components/IDE.vue

```diff
@@ -225,13 +225,38 @@ export default {
 				return true;
 			}
 			try {
+				let isFederated = Utils.size(this.connection.capabilities().toJSON().federation) > 0;
 				let errors = await this.connection.validateProcess(this.process);
 				if (errors.length > 0) {
+					// log entries require a `level` attribute
 					errors.forEach(error => error.level = 'error');
-					this.broadcast('viewLogs', errors, 'Validation Result', 'fa-tasks');
+					// build applicable message and possibly add it to the `errors` object so that it shows up in the logs
+					let toastMessage, logMessage;
+					if(isFederated) {
+						if(Array.isArray(errors['federation:backends'])) {
```

Comment:
```suggestion
						if(Array.isArray(errors['federation:backends']) && errors['federation:backends'].length > 0) {
```

Just to make it a bit more robust in case some backends return bogus.